scripts/build_common.sh

```diff
@@ -0,0 +1,35 @@
+# Builds release profile unless specified other
+PROFILE="--release"
+# Features switch that will be passed to cargo in a docker container
+FEATURES=""
```

Comment:
an example usage would be nice, I'm not quite sure why this is here.

Reply:
I'll add some help. It is meant to parse arguments and return variables such as $PROFILE and $FEATURES so it could be used in `cargo build` calls. It is `source`-ed from build scripts.